src/dashboard/frontend/app/front_page/transfer_browser.html

```diff
@@ -29,17 +29,34 @@
   <div class="help-block">{{ "Access system ID" | translate }}</div>
   </div>
 
-  <div class="col-xs-3">
+  <div class="col-xs-4">
   <button type="submit"
           class="btn btn-default"
           data-toggle="collapse"
           data-target="#transfer_browse_tree"
           aria-expanded="false"
           aria-controls="transfer_browse_tree">{{ "Browse" | translate }}</button>
-  <button type="submit"
+  <div class="btn-group">
+    <button type="submit"
           class="btn btn-success"
           ng-disabled="!vm.enable_submit_button()"
-          ng-click="vm.transfer.start()">{{ "Start transfer" | translate }}</button>
+          ng-click="vm.transfer.start('default')">{{ "Start transfer" | translate }}</button>
+    <button type="submit"
+          class="btn btn-success dropdown-toggle"
+          ng-disabled="!vm.enable_submit_button()"
+          data-toggle="dropdown"
+          aria-haspopup="true"
+          aria-expanded="false">
+      <span class="caret"></span>
+      <span class="sr-only">Toggle Dropdown</span>
+    </button>
+    <ul class="dropdown-menu">
+      <li class="processing-config-choice"
+          ng-repeat="processing_config in vm.transfer.processing_configs">
+        <a ng-click="vm.transfer.start(processing_config)">{{ "Start with" | translate }} "{{ processing_config }}" {{ "config" | translate }}</a>
```

Comment:
I think an interpolated string would be better here:

```
<a ng-click="vm.transfer.start(processing_config)" translate>Start with {{ processing_config }} config</a>
```

You'd get a single translation string:

```
msgid "Start with {{ processing_config }} config"
msgstr ""
```

instead of multiple ones:
```
msgid "Start with"
msgstr ""
...
msgid "config"
msgstr ""
```

which provides better context for translators.

@tw4l Could you please also make sure once this is approved to [extract and update the translation files](https://github.com/artefactual/archivematica/tree/55af1eaa8cd6686903fc72c2b3790bb828f77930/src/dashboard/frontend#translations)? I can help if you need.

Reply:
@replaceafill I'd be happy to update the translation files. Thank you for offering to help! Would it make sense to do that before merging so that the compiled messages are part of the PR?

Reply:
@tw4l Yes, that makes sense. Thanks!